ffi/src/winscard/scard_handle.rs

```diff
@@ -0,0 +1,183 @@
+use std::mem::size_of;
+use std::slice::{from_raw_parts, from_raw_parts_mut};
+
+use ffi_types::winscard::{LpScardIoRequest, ScardContext, ScardHandle, ScardIoRequest};
+use ffi_types::LpCVoid;
+use winscard::winscard::{IoRequest, Protocol, WinScard, WinScardContext};
+use winscard::{Error, ErrorKind, WinScardResult};
+
+/// Represents smart card context handle.
+/// Additionally, it holds allocated buffers and created smart card handles.
+/// We need them because during the smart card context deletion, we need to free all allcated resources.
```

Comment:
nitpick: All the markdown until the first blank line is considered to be a "summary". Generally, we try to keep the summary short. It’s non-blocking.

Reply:
fixed